src/views/statutory/PositionsList.vue

```diff
@@ -186,6 +196,15 @@ export default {
     }),
     prefix () {
       return this.$route.params.prefix
+    },
+    description: function () {
```

Comment:
I propose to set position.body when loading all positions, like this
1) load all positions
2) load all bodies, save it to this.bodies
3) for each position, if there's a body for it and it's in bodies array, run `this.$set(position, 'body', body)`
4) display it from selectedPosition.body.description, and add another <p> to display the message when the body is not set

Reply:
What is really the difference with what it is doing now? Maybe only that now it loads the data multiple times when you go to the same position multiple times, but that is the only thing I can think of.

Reply:
hm, actually yep, it's not that important I'd say